typescript/cli/src/commands/warp.ts

```diff
@@ -60,6 +63,75 @@ export const warpCommand: CommandModule = {
   handler: () => log('Command required'),
 };
 
+export const apply: CommandModuleWithWriteContext<{
+  config: string;
+  symbol?: string;
+  warp: string;
+}> = {
+  command: 'apply',
+  describe: 'Update Warp Route contracts',
+  builder: {
```

Comment:
should we also allow a `--chains` filter so you can deploy to a specific subset of the chains defined in the config? e.g. say you do a deploy across chains but it fails on a particular chain, you can redeploy just on a particular chain

Reply:
I like where you head is. 

I think i'll leave this to a future update if its requested. As of now, if 1 chain fails, and then we run update again, it should skip the other chains because the config diffs will find the successful chains to be the same.

Reply:
ok cool sg